man/man8/zpool.8

```diff
@@ -1739,9 +1741,14 @@ performing this action.
 .It Xo
 .Nm
 .Cm reopen
+.Op Fl n
 .Ar pool
 .Xc
 Reopen all the vdevs associated with the pool.
+.Bl -tag -width Ds
+.It Fl n
+Don't restart the scrub operation if there are devices that need to be
+reslivered.
```

Comment:
How about:

"Do not restart an in-progress scrub operation.  This is not recommended and can
result in partially resilvered devices unless a second scrub is performed."

Reply:
Ok.